lib/feature/home_page/widgets/pokemon_filtered_page.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:pokedex_async_redux/api/models/pokemon_model.dart';
```

Comment:
Filtered page is another feature, so please move it outside the home_page directory